frontend/src/bundles/common/components/sidebar/styles.module.scss

```diff
@@ -48,3 +53,125 @@ a {
         color: var(--white);
     }
 }
+
+// BURGER
+.burgerBackground {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    display: none;
+    width: var(--sidebar-width);
+    min-width: var(--sidebar-width);
+    max-width: var(--sidebar-width);
+    height: var(--header-height);
+    background-color: var(--white);
+    content: "";
+}
+
+.burgerWrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 30;
+    display: none;
+    width: var(--sidebar-width);
+    min-width: var(--sidebar-width);
+    max-width: var(--sidebar-width);
+    height: var(--header-height);
+}
+
+.burgerButton {
+    --size: 20px;
+    --color: var(--text-primary-dark);
+    --g1: linear-gradient(var(--color) 20%, #00000000 0 80%, var(--color) 0)
+        no-repeat content-box border-box;
+    --g2: radial-gradient(
+            circle closest-side at 50% 12.5%,
+            var(--color) 95%,
+            #00000000
+        )
+        repeat-y content-box border-box;
+
+    position: relative;
+    box-sizing: content-box;
+    height: var(--size);
+    padding: 0;
+    color: inherit;
+    font: inherit;
+    background:
+        var(--g2) left var(--p, 0) top,
+        var(--g1) left calc(var(--size) / 10 + var(--p, 0px)) top,
+        var(--g2) right var(--p, 0) top,
```

Comment:
you don't have --p variable here

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/42421305/87770b58-5e8b-49cb-998a-65106e6fd261)
--p  --> --position